v2/workloadapi/client.go

```diff
@@ -412,6 +431,31 @@ func parseX509Bundle(spiffeID string, bundle []byte) (*x509bundle.Bundle, error)
 	return x509bundle.FromX509Authorities(td, certs), nil
 }
 
+// parseJWTSVIDs parses one or all of the SVIDs in the response. If firstOnly
+// is true, then only the first SVID in the response is parsed and returned.
+// Otherwise all SVIDs are parsed and returned.
+func parseJWTSVIDs(resp *workload.JWTSVIDResponse, audience []string, firstOnly bool) ([]*jwtsvid.SVID, error) {
+	n := len(resp.Svids)
+	if firstOnly {
+		n = 1
+	}
+
+	svids := make([]*jwtsvid.SVID, 0, n)
+	for i := 0; i < n; i++ {
+		svid := resp.Svids[i]
+		s, err := jwtsvid.ParseInsecure(svid.Svid, audience)
+		if err != nil {
+			return nil, err
+		}
+		svids = append(svids, s)
+	}
+
+	if len(svids) == 0 {
```

Comment:
This check should move above the for loop and check the length of SVIDs in the response, otherwise, if the response has no SVIDs and `firstOnly` is true, we will panic on line 445.

Reply:
Good point, I directly copied the code of `parseX509SVIDs` and modified it. I didn't notice it here. I think `parseX509SVIDs` also has this problem, so I modified it at the same time. 
In addition, I would like to ask about `java-spiffe`, this  https://github.com/spiffe/java-spiffe/pull/90 not assigned to reviewer, any idea?

Reply:
> I would like to ask about `java-spiffe`, this [spiffe/java-spiffe#90](https://github.com/spiffe/java-spiffe/pull/90) not assigned to reviewer, any idea?

@loveyana I'm reviewing it today.

Reply:
> > I would like to ask about `java-spiffe`, this [spiffe/java-spiffe#90](https://github.com/spiffe/java-spiffe/pull/90) not assigned to reviewer, any idea?
> 
> @loveyana I'm reviewing it today.

In addition, I may need help to look at the https://github.com/HewlettPackard/py-spiffe/pull/105. It seems that the lint in CI on the `py-spiffe` side is wrong, but I passed it locally, and it seems that the previous pr also has the same problem. I hope you can help and review it at the same time